recipe/meta.yaml

```diff
@@ -32,7 +33,8 @@ requirements:
     - python
     - cython >=0.29.24,<3.0
     - llvm-openmp >=4.0.01  # [osx]
-    - numpy
+    - numpy   1.19  # [py==38 or py==39]
+    - numpy   1.21  # [py==310]
     - packaging
```

Comment:
I would simplify these selectors to be `[py<310]` and `[py>=310]`. (This may also be a sign for us to just update the numpy pinning in the aggregate-level cbc.yaml.)